fb_instant_articles.module

```diff
@@ -8,6 +8,38 @@
 use Drupal\Core\Routing\RouteMatchInterface;
 
 /**
+ * No ads will automatically be inserted into Instant Articles.
```

Comment:
@deviantintegral we can, but at this stage of the port there isn't a good home for it yet. Let's leave it for now and come back to it. These constants will most likely see actual use in https://www.drupal.org/node/2872807. There should be a good home in the implementation there. I'd like to wait until that evolves, in the meantime, I'll mark a @ todo.